src/main/resources/application.yml

```
@@ -109,6 +109,14 @@ spring:
               brokers: "\${KAFKA_BINDER_BROKER:}"
               configuration:
                 sasl.jaas.config: "\${KAFKA_PAYMENTS_SASL_JAAS_CONFIG:}"
+        kafka-data-events-producer:
+          type: kafka
+          environment:
+            spring.cloud.stream.kafka.binder:
+              brokers: "\${KAFKA_BINDER_BROKER:}"
```

let's give the possibility to configure through a separate ENV the broker for DATA_EVENTS topic
```suggestion
              brokers: "\${KAFKA_DATA_EVENTS_BINDER_BROKER:\${KAFKA_BINDER_BROKER:}}"
```

let's do the same also for PAYMENTS topic